xrefine1.dfy(64,12): Error: a precondition for this call could not be proved
xrefine1.dfy(49,32): Related location: This is the precondition that might not hold.

Dafny program verifier finished with 3 verified, 1 error
